plugin/references.py

```diff
@@ -30,17 +30,17 @@ class LspSymbolReferencesCommand(LspTextCommand):
 
     capability = 'referencesProvider'
 
-    def __init__(self, view: sublime.View) -> None:
-        super().__init__(view)
-        self._picker = None  # type: Optional[LocationPicker]
-
     def run(self, _: sublime.Edit, event: Optional[dict] = None, point: Optional[int] = None) -> None:
         session = self.best_session(self.capability)
         file_path = self.view.file_name()
         pos = get_position(self.view, event, point)
         if session and file_path and pos is not None:
-            params = text_document_position_params(self.view, pos)
```

Comment:
What was wrong with the old way here though?

Reply:
Nothing, 
but I found the new way more explicit.

Reply:
The return type of was updated from `Dict[str, Any]` to `TextDocumentPositionParams` and since `TextDocumentPositionParams` is a "sealed" object (total=True), you can't add more properties to it in a separate statement.

So with updated return type, doing it like before would be a type error.

Reply:
Does that mean we can't use the `text_document_position_params` function anymore if we want to add more properties to the returned dict?

See https://github.com/sublimelsp/LSP-julia/blob/ad7415e040464afac39796fe07fd58c73b7f9120/plugin.py#L61-L69

Reply:
Correct, that will be a type error now and would need to handle it the same as here.

Reply:
Ok no problem, the alternative handling is quite simple too and I can easily update it.